README.md

```diff
@@ -95,15 +95,15 @@ If you're an experienced Mixed Reality or MRTK developer, check the links in the
 
 Explore MRTK's various types of interactions and UI controls through the example scenes. You can find example scenes under [**Assets/MRTK/Examples/Demos**](/Assets/MixedRealityToolkit.Examples/Demos) folder.
 
-[![Example Scene](Documentation/Images/MRTK_Examples.png)](Documentation/README_HandInteractionExamples.md)
+[![Example Scene](Documentation/Images/MRTK_Examples.png)](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/example-scenes/hand-interaction-examples)
```

Comment:
I'd recommend removing the `en-us` parts of these URLs, to allow peoples' locale preferences to redirect them as needed